Adding Soroban to Stellar Fundamentals (#269)
stellar/stellar-docs: docs/fundamentals-and-concepts/lumens.mdx

sidebar_position: 60
---

> Lumens (XLM) are the native currency of the Stellar network. The lumen is the only token that doesn’t require an issuer or trustline, and it pays all transaction fees, rent, and covers minimum balances on the network.

Reviewer: ```suggestion
Lumens (XLM) are the native currency of the Stellar network. The lumen is the only token that doesn’t require an issuer or trustline, and it is used to pay all transaction fees, rent, and covers minimum balances on the network.
```

To read up on the basics of lumens, head over to our Stellar Learn site: [Stellar Learn: Lumens](https://www.stellar.org/lumens)

## Transaction fees

Stellar requires a small fee for all transactions to prevent ledger spam and prioritize transactions during surge pricing. Transaction fees are paid in lumens.

To learn about fees on Stellar, see our [Fees, Surge Pricing, and Fee Strategies Encyclopedia Entry](../encyclopedia/fees-surge-pricing-fee-strategies).

Smart contract transactions on Stellar employ a different fee structure based on an inclusion fee and resource consumption (which includes [rent](#rent)). Read more in the [Fees and Metering section](https://soroban.stellar.org/docs/fundamentals-and-concepts/fees-and-metering) in the Soroban docs.

## Base reserves

A unit of measurement used to calculate an account’s minimum balance. One base reserve is currently 0.5 XLM.

Validators can vote to change the base reserve, but that’s uncommon and should only happen every few years.

## Rent

> Rent is equivalent to a base reserve for smart contract data. Every smart contract entry must pay rent to exist on the ledger based on how long the entry should live on the ledger before expiration and how big the entry is. There are different rent requirements for each storage type `Persistent`, `Temporary`, and `Instance`, which you can read about in the [Fees and Metering section](https://soroban.stellar.org/docs/fundamentals-and-concepts/fees-and-metering#refundable-resources) of the Soroban docs.

Reviewer 2: ```suggestion
Rent is equivalent to a base reserve for smart contract data. Every smart contract entry must pay rent to exist on the ledger based on how big the entry is and how long the entry can live on the ledger before being archived. There are different rent requirements for each storage type `Persistent`, `Temporary`, and `Instance`, which you can read about in the [Fees and Metering section](https://soroban.stellar.org/docs/fundamentals-and-concepts/fees-and-metering#refundable-resources) of the Soroban docs.
```

> Rent is equivalent to a base reserve for smart contract data. Every smart contract entry must pay rent to exist on the ledger based on how big the entry is and how long the entry can live on the ledger before being archived. There are different rent requirements for each storage type `Persistent`, `Temporary`, and `Instance`, which you can read about in the [Fees and Metering section](https://soroban.stellar.org/docs/fundamentals-and-concepts/fees-and-metering#refundable-resources) of the Soroban docs.

Reviewer: ```suggestion
Smart contract data does not require any base reserves in order to live on the ledger, so every smart contract entry must pay rent instead. The rent charged for an entry to exist on the ledger is based on how big the entry is and how long the it should be live on the ledger before being archived. There are different rent requirements for each storage type `Persistent`, `Temporary`, and `Instance`, which you can read about in the [Fees and Metering section](https://soroban.stellar.org/docs/fundamentals-and-concepts/fees-and-metering#refundable-resources) of the Soroban docs.
```
